google/cloud/storage/tests/slow_reader_integration_test.cc

```diff
@@ -118,6 +118,75 @@ TEST_F(SlowReaderIntegrationTest, StreamingRead) {
   EXPECT_STATUS_OK(status);
 }
 
+TEST_F(SlowReaderIntegrationTest, StreamingReadRestart) {
+  StatusOr<Client> client = Client::CreateDefaultClient();
+  ASSERT_STATUS_OK(client);
+
+  std::string const bucket_name = flag_bucket_name;
+  auto object_name = MakeRandomObjectName();
+  auto file_name = MakeRandomObjectName();
+
+  // Construct a large object, or at least large enough that it is not
+  // downloaded in the first chunk.
```

Comment:
Perhaps just "Construct an object large enough to not be downloaded in the first chunk"?

Reply:
Fixed.